addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1612,6 +1672,9 @@ const actions = {
             localId: composerLocalId,
             attachmentLocalIds: [],
             threadLocalId,
+            textInputContent: '',
+            textInputCursorStart: 0,
+            textInputCursorEnd: 0,
```

Comment:
`text*` before `thread*` (also `localId` is misplaced)

Reply:
Done